specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/expressRouteCrossConnection.json

```diff
@@ -814,10 +812,12 @@
           "description": "The identifier of the circuit traffic."
         },
         "peeringLocation": {
+          "readOnly": true,
           "type": "string",
           "description": "The peering location of the ExpressRoute circuit."
         },
         "bandwidthInMbps": {
+          "readOnly": true,
           "type": "integer",
```

Comment:
These would be considered as breaking change and needs approval from the board.

Reply:
It is approved by ER team.will share the API document reviewed by ER leads